core/src/main/scala/org/apache/spark/sql/delta/commands/DeletionVectorUtils.scala

```diff
@@ -82,6 +83,23 @@ trait DeletionVectorUtils {
     protocol.isFeatureSupported(DeletionVectorsTableFeature) &&
       metadata.format.provider == "parquet" // DVs are only supported on parquet tables.
   }
+
+  /**
+   * Utility method that checks the table has no Deletion Vectors enabled. Deletion vectors
+   * are supported only in read-mode for now. Any updates to tables with deletion vectors
+   * feature are disabled until we add support.
+   */
+  def assertDeletionVectorsNotReadable(
+      spark: SparkSession, metadata: Metadata, protocol: Protocol): Unit = {
+    val disable =
+      Utils.isTesting && // We are in testing and enabled blocking updates on DV tables
+          spark.conf.get(DeltaSQLConf.DELTA_ENABLE_BLOCKING_UPDATES_ON_DV_TABLES)
+    if (!disable && deletionVectorsWritable(protocol, metadata)) {
```

Comment:
Now `assertDeletionVectorsNotReadable` checks that DVs are not writable :D
It should only be checking that DVs are not readable.